mongodbatlas/data_source_mongodbatlas_cluster.go

```diff
@@ -497,6 +498,22 @@ func dataSourceMongoDBAtlasClusterRead(ctx context.Context, d *schema.ResourceDa
 	return nil
 }
 
+func flattenReplicationSpecs(rSpecs []matlas.ReplicationSpec) []map[string]interface{} {
```

Comment:
Moved `flattenReplicationSpecs ` to data source cluster as is so that region mapping logic is isolated to to cluster resource.